nat-lab/tests/telio.py

```diff
@@ -1211,6 +1213,84 @@ async def probe_pmtu(self, host: str, expected: int):
         await self._write_command(["pmtu", host])
         await ev.wait()
 
+    async def get_coredump_pattern(self) -> tuple[str, str]:
```

Comment:
Inisde of CI we are running in our own VM, therefore, maybe it would make sense to simply change the `core_pattern` to something we control?

This way we can control exactly the pattern, and there will be no need to parse it. IMHO it is simpler and more reliable as well.

And when running the tests outside of CI -> we probably can skip coredump collection, as this is something, the one who is running tests can control and take directly from containers.

Reply:
Good point, I've seen that on my other system, unfortunately it's not very helpful if it's not creating a file though. How would you handle that case?

Reply:
> Good point, I've seen that on my other system, unfortunately it's not very helpful if it's not creating a file though. How would you handle that case?

IMHO, we can control `core_pattern` inside of CI and set it to something convenient for us to use. And leave it up to devs outside of CI.

Reply:
TLDR: we can not change the `core_pattern` from within the tests or docker images because of permission issues `sysctl: permission denied on key "kernel.core_pattern"`. Will have another PR for the CI runners.